Use DYLD_LIBRARY_PATH instead of LD_LIBRARY_PATH on OS X

OS X does not use LD_LIBRARY_PATH to define additional loader paths. To find
the wkhtmltox library before installation, DYLD_LIBRARY_PATH needs to be set.

Upstream-Status: Submitted [https://github.com/wkhtmltopdf/wkhtmltopdf/pull/3022]
--- src/pdf/pdf.pro.orig	2016-06-28 18:41:21.000000000 +0200
+++ src/pdf/pdf.pro	2016-06-28 18:43:33.000000000 +0200
@@ -32,6 +32,8 @@
 }
 
 macx {
+    man.extra=DYLD_LIBRARY_PATH=../../bin ../../bin/wkhtmltopdf --manpage | gzip > $(INSTALL_ROOT)$$INSTALLBASE/share/man/man1/wkhtmltopdf.1.gz
+
     CONFIG -= app_bundle
 }
 
